README.md

```diff
@@ -1,77 +1,37 @@
-# discipl-law-reg
+# Discipl Law-Reg [![Build Status](https://travis-ci.org/discipl/discipl-law-reg.svg?branch=master)](https://travis-ci.org/discipl/discipl-law-reg)
 
-Library capable of interpreting published FLINT linked data models of official law and regulation and support self sovereign actors to effectively perform the tasks identified in them through discipl software stack.
+Library capable of interpreting published FLINT models of official law and regulation and support actors with their self-sovereign ids to effectively perform the tasks identified in them through the [Discipl software stack](https://github.com/discipl).
 
-This library creates self sovereign id's in relation to needs automaticly and stores key information at a discipl-core supported platform (probably some local or private distributed wallet as you want to keep this private)
+This library creates self-sovereign ids in relation to needs automatically and stores key information at a discipl-core supported platform (probably some local or private distributed wallet as you want to keep this private)
 
-Enables possibillity to escalate towards an escalation process (convergent facilitation) if any actor does not agree with the automated process and logic or decisions of other actors.
+Enable possibility to escalate towards an escalation process (convergent facilitation) if any actor does not agree with the automated process and logic or decisions of other actors.
```

Comment:
Enables was actually correct here. Maybe clearly would be "This library also enables the possibility...."